lua/rust-tools/dap.lua

```diff
@@ -1,30 +1,25 @@
+local config = require('rust-tools.config')
 local M = {}
 
 function M.setup_adapter()
     local dap = require('dap')
-    dap.adapters.rt_lldb = {
+    dap.adapters.lldb = {
         type = 'executable',
         command = 'lldb-vscode',
-        name = "rt_lldb"
+        name = "lldb"
     }
 end
 
 local function get_cargo_args_from_runnables_args(runnable_args)
     local cargo_args = runnable_args.cargoArgs
 
+    table.insert(cargo_args, '--no-run')
```

Comment:
I made this change because if you try to debug a test which crashes, rust-tools tries to run the binary anyway, and then complains that it crashes. It then launches the debugger anyway.

Reply:
We can't have this, because this only applies to tests. We need a seperate check for the run to change it to build. Check out https://github.com/simrat39/rust-tools.nvim/blob/master/lua/rust-tools/debuggables.lua#L53